integration/configs/browserCompatibilityTest/linux/linux_content_share_join_later_test.config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "`": 1,
+    "name": "linux_content_share_join_later_test.json",
+    "grids": [
+    ],
+    "tests": [
+      {
+        "name": "Content Share Join Later Test on Linux %ts",
```

Comment:
Do we need this `%ts`? I know we have this in current config files but it seems to be useless.

* If you think it's better to keep it, then could you add it to the `name` option of `linux_video_test.config.json` and `linux_audio_test.config.json` file?

* If you think we should get rid of it, then could you remove it from everywhere?

Reply:
I did some research and found something interesting regarding the `%ts`. It turns out we actually might need it since it seems to be used by the KITE-engine to append a timestamp at the end. Take a look at the implementation: https://github.com/webrtc/KITE/blob/1079282b15ef626693ef5247956da2fe8b521172/KITE-Engine/src/main/java/org/webrtc/kite/config/Configurator.java#L83

Also, I found a GitHub issue where one of the users of KITE did not have the `%ts` at the end of the name and KITE engine seems to be complaining about it. https://github.com/webrtc/KITE/issues/127

Reply:
@xuesichao I will advocate for leaving this as it is

Reply:
Cool, nice finding! Thanks for sharing the knowledge 😄